net-im/synapse/synapse-1.80.0.ebuild

```diff
@@ -0,0 +1,205 @@
+# Copyright 2022-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{9..11} )
+DISTUTILS_USE_PEP517=poetry
+
+CRATES="
+	aho-corasick-0.7.19
+	anyhow-1.0.70
+	arc-swap-1.5.1
+	autocfg-1.1.0
+	bitflags-1.3.2
+	blake2-0.10.6
+	block-buffer-0.10.3
+	cfg-if-1.0.0
+	crypto-common-0.1.6
+	digest-0.10.5
+	generic-array-0.14.6
+	hex-0.4.3
+	indoc-1.0.7
+	itoa-1.0.4
+	lazy_static-1.4.0
+	libc-0.2.135
+	lock_api-0.4.9
+	log-0.4.17
+	memchr-2.5.0
+	memoffset-0.6.5
+	once_cell-1.15.0
+	parking_lot-0.12.1
+	parking_lot_core-0.9.3
+	proc-macro2-1.0.52
+	pyo3-0.17.3
+	pyo3-build-config-0.17.3
+	pyo3-ffi-0.17.3
+	pyo3-log-0.8.1
+	pyo3-macros-0.17.3
+	pyo3-macros-backend-0.17.3
+	pythonize-0.17.0
+	quote-1.0.26
+	redox_syscall-0.2.16
+	regex-1.7.1
+	regex-syntax-0.6.27
+	ryu-1.0.11
+	scopeguard-1.1.0
+	serde-1.0.157
+	serde_derive-1.0.157
+	serde_json-1.0.94
+	smallvec-1.10.0
+	subtle-2.4.1
+	syn-1.0.104
+	syn-2.0.2
+	target-lexicon-0.12.4
+	typenum-1.15.0
+	unicode-ident-1.0.5
+	unindent-0.1.10
+	version_check-0.9.4
+	windows-sys-0.36.1
+	windows_aarch64_msvc-0.36.1
+	windows_i686_gnu-0.36.1
+	windows_i686_msvc-0.36.1
+	windows_x86_64_gnu-0.36.1
+	windows_x86_64_msvc-0.36.1
+"
+
+inherit cargo distutils-r1 multiprocessing optfeature systemd
+
+MY_PV="${PV/_rc/rc}"
+
+DESCRIPTION="Reference implementation of Matrix homeserver"
+HOMEPAGE="
+	https://matrix.org/
+	https://github.com/matrix-org/synapse/
+"
+SRC_URI="
+	https://github.com/matrix-org/${PN}/archive/v${MY_PV}.tar.gz
+		-> ${P}.gh.tar.gz
```

Comment:
Let's use `${PN}-${MY_PV}` here as well for consistency. I'd personally declare that as `MY_P=${P/_}` and then use `${PV/_}` inline.

Reply:
I will drop `${MY_PV}` out, it used to have its value when I was starting with this ebuild and I wanted to follow upstream with release candidates, but I don't use them anymore.